adapters/networkHandlersMap.js

```diff
@@ -0,0 +1,9 @@
+const { BqStreamNetworkHandler } = require("../v0/destinations/bqstream/util");
+const { BrazeNetworkHandler } = require("../v0/destinations/braze/util");
+const { MarketoNetworkHandler } = require("../v0/destinations/marketo/util");
+
+module.exports = {
+  braze: BrazeNetworkHandler,
+  marketo: MarketoNetworkHandler,
+  bqstream: BqStreamNetworkHandler
+};
```

Comment:
Instead of a map, how about dynamically `require`ing the module directly in the factory? Additional, overhead for the developer to add an entry here.

Reply:
dynamically requiring wrt to the `destination` name? I think it can be done. The idea here was to maintain a list of the specific handlers in one place.
Another reason why I was trying to avoid dynamic `require` is this --> https://eslint.org/docs/rules/global-require
We currently do it in `versionedRouter` but it really is an anti-pattern, It seems.
WDYT?